dropwizard/service/src/test/java/org/apache/polaris/service/dropwizard/catalog/io/FileIOIntegrationTest.java

```diff
@@ -1,263 +0,0 @@
-import org.apache.polaris.service.dropwizard.test.TestEnvironmentExtension;
-import org.apache.polaris.service.exception.IcebergExceptionMapper;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.Arguments;
-import org.junit.jupiter.params.provider.MethodSource;
-import software.amazon.awssdk.services.s3.model.S3Exception;
-
-/** Collection of File IO integration tests */
-@ExtendWith({
-  DropwizardExtensionsSupport.class,
-  TestEnvironmentExtension.class,
-  PolarisConnectionExtension.class,
-  SnowmanCredentialsExtension.class
-})
-public class FileIOIntegrationTest {
-  private static final DropwizardAppExtension<PolarisApplicationConfig> EXT =
-      new DropwizardAppExtension<>(
-          PolarisApplication.class,
-          ResourceHelpers.resourceFilePath("polaris-server-integrationtest.yml"),
-          ConfigOverride.config(
-              "server.applicationConnectors[0].port",
-              "0"), // Bind to random port to support parallelism
-          ConfigOverride.config("server.adminConnectors[0].port", "0"),
-          ConfigOverride.config("io.factoryType", "test"));
-
-  private static final String catalogBaseLocation = "file:/tmp/buckets/my-bucket/path/to/data";
-  private static TestFileIOFactory ioFactory;
-  private static RESTCatalog restCatalog;
-  private static Table table;
-
-  @BeforeAll
-  public static void beforeAll(
-      PolarisConnectionExtension.PolarisToken adminToken,
-      SnowmanCredentialsExtension.SnowmanCredentials snowmanCredentials,
-      @PolarisRealm String realm) {
-    ioFactory = (TestFileIOFactory) EXT.getConfiguration().findService(FileIOFactory.class);
-
-    FileStorageConfigInfo storageConfigInfo =
-        FileStorageConfigInfo.builder()
-            .setStorageType(StorageConfigInfo.StorageTypeEnum.FILE)
-            .setAllowedLocations(List.of(catalogBaseLocation))
-            .build();
-    Catalog catalog =
-        PolarisCatalog.builder()
-            .setType(Catalog.TypeEnum.INTERNAL)
-            .setName("mycatalog")
-            .setProperties(
-                org.apache.polaris.core.admin.model.CatalogProperties.builder(catalogBaseLocation)
-                    .build())
-            .setStorageConfigInfo(storageConfigInfo)
-            .build();
-
-    restCatalog =
-        TestUtil.createSnowmanManagedCatalog(
-            EXT, adminToken, snowmanCredentials, realm, catalog, Map.of());
-
-    Namespace namespace = Namespace.of("myns");
-    restCatalog.createNamespace(namespace);
-
-    String tableName = "mytable";
-    table =
-        restCatalog
-            .buildTable(
-                TableIdentifier.of(namespace, tableName),
-                new Schema(required(3, "id", Types.IntegerType.get(), "mydoc")))
-            .create();
-  }
-
-  @Test
-  void testGetLengthExceptionSupplier() {
```

Comment:
This test case has been dropped completely, but in reality it used to test just other test code :shrug: